rotkehlchen/externalapis/etherscan.py

```diff
@@ -200,6 +211,9 @@ def _query(
             try:
                 result = json_ret.get('result', None)
                 if result is None:
+                    if action in ('eth_getTransactionByHash', 'eth_getTransactionReceipt'):
```

Comment:
Hmm why to silence the error in these cases?

Reply:
Because if a transaction does not exist on-chain, Etherscan returns `{'result': null}` as a response

Reply:
Yeah, but should then a Remote Error / Input Error be raised?

Reply:
`RemoteError` would be caught `self._query()` of `EVMNodeInquirer` which is generic